gettsim/parameters/ges_rente.yaml

```diff
@@ -739,7 +750,8 @@ regelaltersgrenze:
   reference: § 35 Satz 2 SGB VI
   note:
     de: >-
-      Diese Funktion ist hier ab 2002 angegeben. Analog zur implementierung des Steuersystems. Bis einschließlich Jahrgang 1946 ist NRA 65, danach steigt es bis Jahrgang 1964.
+      Diese Funktion ist hier ab 2002 angegeben. Vor 2002 ist das Steuersystem nicht
```

Comment:
I don't get this sentence. What does it have to do with tax system being implemented or not?

This particular function just was not relevant before, wasn't it?

Reply:
the first version was already in there, I guess it was copied and pasted from some other thresholds. In principle the threshold is need at all times at which we want to calculate the pensions. I changed the scope of this function to 1984 -  the first survey year of the soep. I checked that going back to cohort 1924 the threshold 65 is still correct.